contracts/handlers/ERC721Handler.sol

```diff
@@ -4,24 +4,31 @@ pragma experimental ABIEncoderV2;
 import "../ERC721Safe.sol";
 import "../interfaces/IDepositHandler.sol";
 import "../erc/ERC721/ERC721Mintable.sol";
+import "../interfaces/IBridge.sol";
 
 contract ERC721Handler is IDepositHandler, ERC721Safe {
```

Comment:
This handler should include the same whitelisting code as the ERC20 handler - we could probably make this a separate issue though